src/CLIMA.jl

```diff
@@ -33,5 +33,6 @@ include("Atmos/Model/AtmosModel.jl")
 include("Diagnostics/Diagnostics.jl")
 include("InputOutput/VTK/VTK.jl")
 include("Driver/Driver.jl")
+include("Courant/Courant_presets.jl")
```

Comment:
Should this be here or even have a module itself? These implementations seem to be `AtmosModel` specific -- would there be `OceanModel` equivalents? If so, the functions should be declared in `DGModel`. Atmos-specific methods should be within `Atmos/` somewhere.

Reply:
Agreed, move this under `Atmos/Model/courant.jl`. If we find there are common parts for Ocean, we can move them out later.

Reply:
The courant functions use mainly Balance law structures so they should be usable with the Ocean model assuming the same turbulence models are used.

Reply:
Revised per reviewer recommendation.